packages/common/infra/src/blocksuite/migration/fixing.ts

```diff
@@ -13,6 +13,10 @@ export function fixWorkspaceVersion(rootDoc: YDoc) {
    * Blocksuite just set the value, do nothing else.
    */
   function doFix() {
+    if (meta.size === 0) {
```

Comment:
not sure if it is better to use doc.whenSynced flag

Reply:
I added a new isRootDocLoaded flag on syncEngine, should be better